example/lib/examples/builder_constructor_cds_example.dart

```diff
@@ -0,0 +1,117 @@
+import 'dart:math' as math;
+
+import 'package:animated_book_list/animated_book_list.dart';
+import 'package:flutter/material.dart';
+
+///Creating a class object
+class Cd {
+  String cdCoverImgUrl;
+  String cdDiscImgUrl;
+  List<Color> cdGradient;
+
+  Cd(
+      {required this.cdCoverImgUrl,
+      required this.cdDiscImgUrl,
+      required this.cdGradient});
+}
+
+///Simulates a background query with response a List of objects
+List<Cd> cds = [
+  Cd(
+    cdCoverImgUrl:
+        'https://m.media-amazon.com/images/I/81ogsUqshzL._SX425_.jpg',
+    cdDiscImgUrl:
+        'https://www.theaudiodb.com/images/media/album/cdart/rtvxpt1429303647.png',
+    cdGradient: [Colors.black, Colors.black45],
+  ),
+  Cd(
+    cdCoverImgUrl:
+        'https://m.media-amazon.com/images/I/61OBOiVbkeL._SX425_.jpg',
+    cdDiscImgUrl:
+        'https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEgyYQF3KPf68-hsZXiqpr0TJboii0XhbKMi1aD7423DpcqTszjech6cBfqFEuSM7qBUQS-_TjZMMCafWlCJOZfqkzRVfuNSCOL9BQSu8L94hiYW33JWp37YCX684YlOGC-kyi6FDloHXBwpTOApv9Fia69H_vTMVzUSrZeFU5_0YedwBM7GzrsUvXYz4A/s1600/1973%20The%20Dark%20Side%20Of%20The%20Moon%20(C.D%20E.U%2030th%20SACD)%20(4B).png',
+    cdGradient: [
+      Colors.red,
+      Colors.orange,
+      Colors.yellow,
+      Colors.green,
+      Colors.blue,
+      Colors.purple
+    ],
+  ),
+  Cd(
+    cdCoverImgUrl:
+        'https://m.media-amazon.com/images/I/91G5ndi8yFL._SX425_.jpg',
+    cdDiscImgUrl:
+        'https://d2bzx2vuetkzse.cloudfront.net/unshoppable_producs/3f3ec57e-80d2-4f73-a2a5-6bbeeeaf72f7.png',
+    cdGradient: [Colors.black, Colors.grey, Colors.blue, Colors.black],
+  )
+];
+
+///Widget class example
+class BuilderConstructorCdsExample extends StatelessWidget {
+  final bool horizontalView;
+
+  BuilderConstructorCdsExample({
+    super.key,
+    required this.horizontalView,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    ///List of all objects
+    return AnimatedBookList.builder(
+      ///Scroll axis
+      scrollDirection: horizontalView ? Axis.horizontal : Axis.vertical,
+
+      ///Number of items on the list
+      itemCount: cds.length,
+
+      //Item constructor
+      itemBuilder: (_, index) {
+        ///For each object we return a widget constructor with the data.
+        return AnimatedBookWidget.builder(
+          ///Size parameter
+          size: horizontalView ? Size.fromWidth(225) : Size.fromHeight(225),
```

Comment:
Values such as the size of the widgets (Size.fromWidth(225), Size.fromHeight(225)) could be defined as constants at the beginning of your file to facilitate global adjustments and improve readability.

Reply:
Correct!